tests/DownloadTest.php

```diff
@@ -50,20 +50,50 @@ protected static function execute($command)
         ];
     }
 
+    protected static function executeTCI(array $params)
+    {
+        return static::execute(implode(' ', array_merge([
+            'php',
+            escapeshellarg(__DIR__ . '/../tiny-composer-installer.php'),
+        ], $params)));
+    }
+
     /**
      * @medium
      */
     public function testTinyAndOfficialDownloadTheSameComposer()
     {
         $tinyFile = tempnam(sys_get_temp_dir(), 'tiny-composer-');
-        $result = static::execute(implode(' ', [
-            'php',
-            escapeshellarg(__DIR__ . '/../tiny-composer-installer.php'),
-            escapeshellarg($tinyFile),
-        ]));
+        $result = static::executeTCI([escapeshellarg($tinyFile)]);
```

Comment:
Why escape them before passing into the function? Would make more sense to map `escapeshellarg` onto `$params` in `executeTCI` imo.

Reply:
I was about to say “well, but then I can’t pass non-escaped arguments to `executeTCI()` when I have to”. But seeing that [I _forgot_ to escape an argument](https://github.com/fastbill/tiny-composer-installer/blob/09822b3bf0844e51d63f99ffc6d1d105c49573ec/tests/DownloadTest.php#L93), I’ll change it like you suggested.